app/models/pixel.rb

```diff
@@ -49,7 +49,7 @@ class << self
   # Records an conversion for the passed impression_id
   # This operation is a noop if no impression is found
   def record_conversion(impression_id, test: false, metadata: {})
-    impression = Impression.find_by(id: impression_id, campaign_id: Campaign.where(organization_id: organization_id).select(:id))
```

Comment:
I really wasn't sure the decision behind finding the impression this way so there is a complete possibility that my change is not correct. However, this does work to correctly pull the impression id and org id and find the impression.